lib/OpenAPI/Client.pm

```diff
@@ -188,6 +199,30 @@ sub _build_tx {
     }
   }
 
+  # Handling requestBody for OpenAPIv3
+  if ($op_spec->{requestBody}) {
+    my $content_key = [keys %{$op_spec->{requestBody}{content}}]->[0];
+    my $ct          = {'application/x-www-form-urlencoded' => 'form', 'application/json' => 'json'}->{$content_key};
```

Comment:
I'm not sure if I like this line two much. Could you use a ternary operator instead? And maybe default to "application/json"?

Reply:
Addressing this in the next push